lib/core/seleniumRunner.js

```diff
@@ -551,21 +558,48 @@ class SeleniumRunner {
       // the function is not null and we want to run
       // the function.
       if (!script) {
-        let func = category[scriptName].function;
+        let func = category[scriptName][type];
         if (!func) {
-          throw 'Function and script cannot both be null in ' +
-            scriptName +
-            '.';
+          log.info(
+            'Skipping ' + type + ' function invocation in ' + scriptName + '.'
+          );
+          continue;
         }
+        log.info(
+          'Executing ' + type + ' function invocation in ' + scriptName + '.'
+        );
         // We wrap the source code of the function in parenthesis
         // "(...)" to contain it in a separate scope. We add a
-        // "();" to the source of the function to do the actual
+        // "()" to the source of the function to do the actual
         // invocation. The script writer cannot do this in their
         // source because it would force the evaluation of the
         // function too soon and cause undefined reference errors.
         script = '( ' + func + ' )()';
         requires = category[scriptName].requires;
-        isAsyncOverride = category[scriptName].isAsync;
+        isAsyncOverride =
```

Comment:
How is this an override?

Reply:
If it is `true`, then it overrides a `false` value in the function's `isAsync` parameter. Does that seem reasonable?